runtime/doc/autocmd.txt

```diff
@@ -1302,18 +1302,17 @@ TermResponse			After the response to |t_RV| is received from
 				takes time is involved.
 							*TermResponseAll*
 TermResponseAll			After the response to |t_RV|, |t_RC|, |t_RS|,
-				|t_RB|, |t_RF|, or |t_u7| are received from
+				|t_u7| or any OSC command are received from
 				the terminal.  The value of |v:termresponse|,
 				|v:termblinkresp|, |v:termstyleresp|,
-				|v:termrbgresp|, |v:termrfgresp|, and
-				|v:termu7resp|, correspondingly, can be used.
-				<amatch> will be set to any of:
+				|v:termu7resp|, and |v:termosc|
+				correspondingly, can be used.  <amatch> will
+				be set to any of:
 				    "version",
 				    "cursorblink",
 				    "cursorshape",
-				    "background",
-				    "foreground",
 				    "ambiguouswidth"
+				    "osc"
```

Comment:
Note: Traditionally, TermResponseAll was also used for "foreground" and "background" patterns. These are now handled as part of the "osc" value. For backwards compatibility, however, the colorresp.vim plugin will handle "osc" events and emit TermResponseAll autocommand events when it encounters "foreground" and "background" values.